lib/next_ls/runtime.ex

```diff
@@ -120,7 +120,7 @@ defmodule NextLS.Runtime do
                 NextLS.Logger.log(logger, "The runtime for #{name} has successfully shutdown.")
 
               reason ->
-                NextLS.Logger.error(logger, "The runtime for #{name} has crashed with reason: #{reason}.")
+                NextLS.Logger.error(logger, "The runtime for #{name} has crashed with reason: #{inspect(reason)}.")
```

Comment:
This fixes a `String.Chars` protocol error when the reason is a tuple, for example.